app/views/content/train-to-be-a-teacher/promos/_adviser-promo-career-changers.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
   <% promo.left_section(classes: %w[tta-background]) %>
   <% promo.right_section(heading: "Get free one-to-one support") do %>
-      <p>An adviser can talk you through the support available if you're changing career and answer all your questions about getting into teaching.</p>
+      <p>Do not let doubts of changing career hold you back – our experienced teaching advisers can support you through the switch. Chat to an adviser through email, text, or phone.</p>
```

Comment:
```suggestion
      <p>Don't let doubts of changing career hold you back – our experienced teaching advisers can support you through the switch. Chat to an adviser through email, text, or phone.</p>
```
Not sure about this @EllieNodder - I like the second bit about 'our experienced teaching advisers.....' but wonder about the doubts bit, as there will be a lot of considerations (financial, family, etc.)?

Reply:
this was one that I changed to match up with the key messages Naeem sent across but happy to tweak!

Reply:
Happy for you to take a call on this :-) Keisha might have some useful insight?